location.md

```diff
@@ -14,6 +14,28 @@
 
 ### Selection
 
-Have a selection of dates available that you can use for choosing locations.
+Have a selection of dates available that you can use for choosing
+locations.
 
 ## Locations
+
+Depending on the conference setup and type your needs for a location
+will change.
```

Comment:
you have some weird line breaks here and in the paragraph below. Can you remove them?